lib/elixir/lib/calendar/iso.ex

```diff
@@ -207,7 +207,6 @@ defmodule Calendar.ISO do
 
   # Converts count of days since 0000-01-01 to {year, month, day} tuple.
   @doc false
-  @doc since: "1.5.0"
```

Comment:
I'm not sure if this comment is out of the scope of this PR 😅  

But, there are other cases within the same module with `@doc false` and `@doc since: "1.5.0` :)

Reply:
Good catch. I have done in a separate commit here but also pushed to master. So we should be see an empty diff for that file here now. :)